packages/entity-server/src/routes/hierarchy/types.ts

```diff
@@ -20,7 +20,21 @@ export interface MultiEntityRequestParams {
 export type RequestBody = Record<string, never>;
 
 export type MultiEntityRequestBody = RequestBody & {
-  entities: string[] | undefined;
+  entities: string[];
+};
+
+export const MultiEntityRequestBodySchema = {
+  type: 'object',
+  properties: {
+    entities: {
+      type: 'array',
+      items: {
+        type: 'string',
```

Comment:
Had a hard to debug error come out because the entities array was provided, but the value was null instead of a string. Swapped to json schema to improve error messaging.